### DISCUSSION TITLE
Guidance Needed: Reverting an Accidental Merge of a Pull Request

### DISCUSSION THREAD

**Discussion by Tanzimhossain222** [community/community#68977] [New to GitHub]
Hello Community,

I accidentally merged a pull request and I'm now facing some difficulties. Can anyone guide me on how to reverse this action? Any suggestions or advice would be greatly appreciated.

Thanks in advance.

**Comment by 0xTanzim** [comment]
1. **Identify the Commit**: Find the commit that was created when you merged the pull request. You can do this by checking the Git log or using tools like GitHub's commit history.

2. **Copy the Commit Hash**: Copy the commit hash of the merge commit that you want to undo. It typically looks like a long string of characters, e.g., `abcdef12345`.

3. **Revert the Merge**: Run the following command in your terminal, replacing `commit-hash` with the actual commit hash you copied:

   ```
   git revert -m 1 commit-hash
   ```

   This command creates a new commit that undoes the changes introduced by the merge.

4. **Push to Remote**: After successfully reverting the merge locally, push the changes to your remote repository:

   ```
   git push origin master
   ```

   Replace `master` with your branch name if you're not working on the master branch.

5. **Verify**: Double-check your repository on the Git hosting platform (e.g., GitHub) to ensure that the merge has been undone and the changes are reflected.

**Reply by Tanzimhossain222** [reply]
Thanks

**Reply by 0xTanzim** [reply]
💕

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
